### ISSUE TITLE
Keras Accuracy stuck at 50-60%

### ISSUE THREAD

**Issue by redincer** [keras-team/keras#15727]
Hi, i'm working on ISCX dataset which contais Network data Vpn or NonVpn. Then, I wanna use DeepLearning classification which is Vpn and which is Non-Vpn data. 
This is my dataset.
![image](https://user-images.githubusercontent.com/17345114/144054657-8e0af1fd-e2a0-48bc-b617-ad15bef61f9c.png)
![image](https://user-images.githubusercontent.com/17345114/144055431-1896a594-1d4b-490a-9a99-fc7557310d7b.png)
Then I  choose 8965 Vpn and 8965 Non-Vpn data from my dataset.
Then I converted my data to range 0-1
![image](https://user-images.githubusercontent.com/17345114/144055003-969af955-83a7-4c60-84d0-71374535db6f.png)
X_train,X_test,y_train,y_test=train_test_split(X,y,test_size=0.2,random_state=42)
![image](https://user-images.githubusercontent.com/17345114/144055113-cfd604c2-9d2e-489b-8e63-80dd4b6f7c74.png)
![image](https://user-images.githubusercontent.com/17345114/144055140-e154aa24-62cc-405b-9ef6-9f0591ff7212.png)
![image](https://user-images.githubusercontent.com/17345114/144055172-68024276-9849-404d-88d5-7ffc6cf62057.png)

I can't take the accuacy over 60&

**Comment by chunduriv** [comment]
@redincer, As we can see here your model is very simple. Please can you add more layers, try with different batch_size, try with different activation functions and apply data augmentation to achieve a better result. 

Could you please take a look at this [issue](https://stackoverflow.com/questions/66988763/keras-model-accuracy-is-not-improving-image-classification), [issue2](https://datascience.stackexchange.com/questions/57171/how-to-improve-low-accuracy-keras-model-design) which discusses about the similar issue and let us know if it helps? Thanks!

**Comment by innat** [comment]
@redincer 
If your problem is binary classification, then fix your last layer's node and activation functions 

```
Dense(1, activations='sigmoid')
```

If you're problem is multi-class classification, then fix your loss function. It should be categorical cross entropy instead of binary.

**Comment by redincer** [comment]
@innat  my problem is multi-class classification. I'm trying to categorize the daha which is Vpn or Non-Npn data. My output can be vpn or non-vpn. I tried categorical cross entropy activaotin instead of binary. But the accuracy didn't increase.

**Comment by redincer** [comment]
@chunduriv  I added more layers with different combination but i didn't take accuracy more than 61%.

**Comment by chunduriv** [comment]
@redincer, You should not use `categorical cross entropy` as a loss since your usecase is binary classification.

Did you tried to change last layer as suggested by @innat  `Dense(1, activations='sigmoid')` and let us know? Thanks!
